lib/node-token-validation/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@azure/node-token-validation",
+  "author": {
+    "name": "Microsoft",
+    "email": "nugetaad@microsoft.com",
+    "url": "https://www.microsoft.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
+  },
+  "version": "1.0.0-alpha.0",
+  "description": "Token validation library written in Node.js",
+  "keywords": [
+    "js",
+    "AAD",
+    "msal",
+    "oauth",
+    "validation",
+    "tokens"
+  ],
+  "sideEffects": false,
+  "main": "./dist/index.cjs.js",
+  "engines": {
+    "node": ">=0.8.0"
```

Should this be 12+?

---

I would say even versions about 12:

```suggestion
    "node": "12 || 14 || 16"
```